.circleci/config.yml

```diff
@@ -1588,4 +1588,4 @@ workflows:
           requires:
             - approve thorchain coinstack (taxistake)
           <<: *thorchain
-          <<: *only-main
\ No newline at end of file
+          <<: *only-main
```

Comment:
revert EOF?

Reply:
not a huge deal, let's get it in so we can get thorchain back online